File: clients/src/main/java/org/yigitcanyontem/clients/users/profile/UsersProfileCreateDto.java
package org.yigitcanyontem.clients.users.profile;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UsersProfileCreateDto {
    private String firstName;
    private String lastName;
    private String profilePictureUrl;
    private String bannerPictureUrl;
    private String bio;
    private String city;
    private String country;
    private String website;
    private String jobTitle;
    private Date birthDate;
    private Date createdAt;
    private String linkedinProfile;
    private String githubProfile;
    private String mediumProfile;
}

File: frontend/codeConnect/src/models/user/UsersProfileCreateDto.ts
export interface UsersProfileCreateDto {
    firstName: string;
    lastName: string;
    profilePictureUrl: string;
    bannerPictureUrl: string;
    bio: string;
    city: string;
    country: string;
    website: string;
    jobTitle: string;
    birthDate: Date;
    createdAt: Date;
    linkedinProfile: string;
    githubProfile: string;
    mediumProfile: string;
}